lib/apiservers/engine/backends/container.go

```diff
@@ -1605,6 +1600,70 @@ func (c *Container) containerAttach(name string, ca *backend.ContainerAttachConf
 	return nil
 }
 
+// ContainerRename changes the name of a container, using the oldName
+// to find the container. An error is returned if newName is already
+// reserved.
+func (c *Container) ContainerRename(oldName, newName string) error {
+	defer trace.End(trace.Begin(newName))
+
+	if oldName == "" || newName == "" {
+		err := fmt.Errorf("neither old nor new names may be empty")
+		log.Errorf("%s", err.Error())
+		return derr.NewErrorWithStatusCode(err, http.StatusInternalServerError)
+	}
+
```

Comment:
Can you add a log.Info line which includes the old name being renamed to the new name?

`log.Infof("Container %s renamed to %s", oldname, newname)`